client/lib/models/theme.dart

```diff
@@ -1,39 +1,68 @@
 import 'package:flutter/material.dart';
 import 'package:flutter_secure_storage/flutter_secure_storage.dart';
 
+class ColorTypes {
+  Color? main;
+  Color? light;
+  Color? dark;
+
+  ColorTypes({this.main, this.light, this.dark});
+}
+
 class AppColors {
-  static Color primaryColor = const Color.fromRGBO(103, 80, 164, 1);
-  static Color primaryAccent = const Color.fromRGBO(79, 55, 139, 1);
-  static Color textColor = const Color.fromRGBO(29, 27, 32, 1);
-  static Color error = const Color.fromRGBO(179, 38, 30, 1);
+  static ColorTypes primaryColor = ColorTypes(
+    main: const Color.fromRGBO(103, 80, 164, 1),
+    light: const Color.fromRGBO(139, 116, 203, 1),
+    dark: const Color.fromRGBO(79, 55, 139, 1),
+  );
+
+  static ColorTypes error = ColorTypes(
+      main: const Color.fromRGBO(208, 68, 65, 1),
```

Comment:
Add comments near this colors or at least assign them to constant to make it easy to know, what is this color exactly